doc/users/next_whats_new/2017-11-1_figure_align_labels.rst

```diff
@@ -0,0 +1,35 @@
+xlabels and ylabels can now be automatically aligned
+----------------------------------------------------
+
+Subplot axes ``ylabels`` can not line up horizontally if the tick labels
+are very different widths.  The same can happen to ``xlabels`` if the
+ticklabels are rotated on one subplot (for instance).  The new methods
+on the ``Figure`` class: ``fig.align_xlabels()`` and ``fig.align_ylabels()``
```

Comment:
rst syntax police here:
```
on the `Figure` class: `Figure.align_xlabels` and `Figure.align_ylabels`
```
or
```
on the `Figure` class: `~Figure.align_xlabels` and `~Figure.align_ylabels`
```
These are refering to things that have their own entries in the docs (the class and the method.  The tilde is documented at http://www.sphinx-doc.org/en/stable/domains.html#role-py:obj and quite useful.